x-pack/plugins/search_indices/public/plugin.ts

```diff
@@ -65,8 +65,17 @@ export class SearchIndicesPlugin
     };
   }
 
-  public start(core: CoreStart): SearchIndicesPluginStart {
+  public start(
+    core: CoreStart,
+    deps: SearchIndicesAppPluginStartDependencies
+  ): SearchIndicesPluginStart {
+    const { indexManagement } = deps;
     docLinks.setDocLinks(core.docLinks.links);
+    indexManagement?.extensionsService.setIndexDetailsPageRoute({
+      renderRoute: (indexName) => {
+        return `/app/elasticsearch/indices/index_details/${indexName}`;
```

Comment:
I'm worried this static URL could be a little brittle.

What if we create a URL locator in the index management plugin that can be leveraged?

There was a similar request opened recently for another area of Index Management - https://github.com/elastic/kibana/issues/195083

Reply:
`search_indices` has a [dependency](https://github.com/elastic/kibana/blob/main/x-pack/plugins/search_indices/kibana.jsonc#L15) on `index_managment` plugin. Adding a URL locator would create a loose dependency of search_indices in index management plugin and cause circular dependency. 

Agreed static URL could be a bit fragile, but I think its a better option to avoid the circular dependency. WDYT?

Reply:
You should be able to avoid circular dependencies as you can get them from the central registry, via `share` plugin.

Reply:
If we use a locator we will also need to gate usage of the locator in `index_management` based on the solution. Currently this is only used in serverless, but in 8.17 time frame we will also need this in stateful when it's the search solution nav. And possibly in some cases for the classic nav. 

By using the extension service that conditional logic can be handled by `search_indices` instead of `index_management`.